addons/account/static/src/components/char_with_placeholder_field/char_with_placeholder_field.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { exprToBoolean } from "@web/core/utils/strings";
+import { registry } from "@web/core/registry";
+import { CharField, charField } from "@web/views/fields/char/char_field";
+import { formatChar } from "@web/views/fields/formatters";
+
+export class CharWithPlaceholderField extends CharField {
+    static template = "account.CharWithPlaceholderField";
+    static props = {
+        ...CharField.props,
+        placeholderField: { type: String },
```

Comment:
@aab-odoo 
Should we keep this custom, or can we add the option to `web`?
I feel like this option could be useful elsewhere, and this is a lot of boilerplate code for not much

Reply:
@william-andre @antoine162 we checked with cth and we can't find any other usecase for this, so we'll prefer this to be kept as a custo in account